js/packages/cli/src/commands/upload.ts

```diff
@@ -370,22 +370,25 @@ export async function uploadV2({
     saveCache(cacheName, env, cacheContent);
   }
 
-  let uploadSuccessful = true;
+  let uploadSuccessful = false;
   if (!hiddenSettings) {
-    uploadSuccessful = await writeIndices({
-      anchorProgram,
-      cacheContent,
-      cacheName,
-      env,
-      candyMachine,
-      walletKeyPair,
-      rateLimit,
-    });
+    while (!uploadSuccessful) {
+      console.log(`Starting upload at ${new Date()}`);
```

Comment:
I would not call this an upload, probably good to call it "Starting indices writing at"